CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## unreleased
 
+### Changed
+
+- Updated `axum` to v0.6.0.
+
 ## Removed
```

Comment:
Shouldn't this be `###` instead of `##`?

Reply:
it should, yes 😅